mule-extensions-api/src/main/java/org/mule/runtime/extension/api/metadata/ComponentMetadataConfigurer.java

```diff
@@ -88,6 +90,14 @@ ComponentMetadataConfigurer setKeysResolver(TypeKeysResolver keysResolver, Strin
    */
   ComponentMetadataConfigurer setChainInputTypeResolver(ChainInputTypeResolver chainInputTypeResolver);
 
+  /**
+   * Convenience method to configure a scope whose inner chain/s will receive the same input type of the scope itself. An example
```

Comment:
`whose` -> `which`

Reply:
I think _whose_ is correct for inanimate antecedents too: https://en.wikipedia.org/wiki/Inanimate_whose

If you prefer I could go with "...to configure a scope _of which the_ inner chain will..."
or "...to configure a scope _the inner chain of which_ will..."

But "...to configure a scope _which_ inner chain will..." doesn't seem correct.

Reply:
Updated, let me know what you think.